### PR TITLE
23 yamcs config

### PR DESCRIPTION
**Pull request by clogan-WHL**
Issue 23 in sqweakyweasle
Configure yamcs for RTS/CTS and baud rate.  Requires updating to yamcs.1.2.0.jar.

### CODE REVIEW

File: config/fixedwing/gemini2/cpd/target/cfe_es_startup.scr [hunk 1]
```diff
@@ -20,13 +20,10 @@ CFE_APP, /embedded/MD.so,        MD_AppMain,      MD,       130, 32768, 0x0, 0,
 CFE_APP, /embedded/SC.so,        SC_AppMain,      SC,       120, 32768, 0x0, 0, 0;
 CFE_APP, /embedded/ZUSUP.so,     ZUSUP_AppMain,   ZUSUP,    240, 32768, 0x0, 0, 0;
 CFE_APP, /embedded/ICM20689.so,  ICM20689_AppMain, ICM20689, 48, 32768, 0x0, 0, 2;
-CFE_APP, /embedded/HMC5883.so,   HMC5883_AppMain, HMC5883,   53, 32768, 0x0, 0, 2;
 CFE_APP, /embedded/MS5611.so,    MS5611_AppMain,  MS5611,    57, 32768, 0x0, 0, 2;
 CFE_APP, /embedded/RCIN.so,      RCIN_AppMain,    RCIN,      63, 32768, 0x0, 0, 2;
-CFE_APP, /embedded/GPS.so,       GPS_AppMain,     GPS,       67, 32768, 0x0, 0, 2;
 CFE_APP, /embedded/CI.so,        CI_AppMain,      CI,      115, 32768, 0x0, 0, 0;
 CFE_APP, /embedded/TO.so,        TO_AppMain,      TO,        124, 32768, 0x0, 0, 2;
-CFE_APP, /embedded/ASPD4525.so,  ASPD4525_AppMain,ASPD4525,  56, 32768, 0x0, 0, 2;
```
Comment (lorenzo-gomez-windhover) [inline config/fixedwing/gemini2/cpd/target/cfe_es_startup.scr]: We need these apps for flight. We should add them back to the startup script.

File: config/fixedwing/gemini2/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml [hunk 2]
```diff
@@ -74,40 +76,46 @@ dataLinks:
         epoch: CUSTOM
         epochUTC: "1970-01-01T00:00:00Z"
         timeIncludesLeapSeconds: false
-         
-  - name: tc_cpd
```
Comment (lorenzo-gomez-windhover) [inline config/fixedwing/gemini2/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml]: Sorry this is on me: undo the changes to this file. Sorry about that; I misled you.

File: config/multirotor/quad/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml [hunk 3]
```diff
@@ -74,40 +76,46 @@ dataLinks:
...
-        timeIncludesLeapSeconds: false
         
+  - name: tmtc_cpd
+    class: com.windhoverlabs.yamcs.tctm.SerialATTmTcDatalink
+    device: /dev/ttyUSB0
+    baudRate: 115200
+    flowControl: RTS_CTS
+    tm_config:
+      stream: tm_realtime
+      packetInputStreamClassName: com.windhoverlabs.yamcs.tctm.RFC1055PacketInputStream
```
Comment (lorenzo-gomez-windhover) [inline config/multirotor/quad/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml]: Sorry this is on me: undo the changes to this file. Sorry about that; I misled you.

**Review by lorenzo-gomez-windhover — CHANGES_REQUESTED** [review]

File: config/multirotor/quad/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml (hunk 3 above)
Comment (lorenzo-gomez-windhover) [reply config/multirotor/quad/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml]: Fixed in 5c7b4c86c84f0f467a0fd2d0b1c8612990dc0cd7.
Thanks!

File: config/fixedwing/gemini2/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml (hunk 2 above)
Comment (lorenzo-gomez-windhover) [reply config/fixedwing/gemini2/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml]: Fixed in https://github.com/WindhoverLabs/airliner/commit/5c7b4c86c84f0f467a0fd2d0b1c8612990dc0cd7.
Thanks!

File: config/fixedwing/gemini2/cpd/target/cfe_es_startup.scr (hunk 1 above)
Comment (clogan-WHL) [reply config/fixedwing/gemini2/cpd/target/cfe_es_startup.scr]: OK, I didn't mean to commit this hear. Fixed.

File: config/fixedwing/gemini2/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml (hunk 2 above)
Comment (clogan-WHL) [reply config/fixedwing/gemini2/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml]: Fixed.

File: config/multirotor/quad/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml (hunk 3 above)
Comment (clogan-WHL) [reply config/multirotor/quad/ppd/sitl/server_overlay/etc/yamcs.fsw.yaml]: Fixed.

File: config/fixedwing/gemini2/cpd/target/cfe_es_startup.scr (hunk 1 above)
Comment (lorenzo-gomez-windhover) [reply config/fixedwing/gemini2/cpd/target/cfe_es_startup.scr]: Thanks!

**Review by lorenzo-gomez-windhover — APPROVED** [review]
LGTM!

**Comment by clogan-WHL** [comment]
See 23_2 yamcs config instead.
